api.py

```
@@ -0,0 +1,1082 @@
+import argparse
+import asyncio
+import json
+import logging
+import os
+import random  # Add import for random selection
+import signal
```

Ruff (0.8.2)

`signal` imported but unused

Remove unused import: `signal`

(F401)